components/IconToggle/styles.scss

```diff
@@ -0,0 +1,39 @@
+@import "base-mixins/_all.scss";
```

Comment:
Do we not want to make this an available rails pattern? Putting the styles here makes it not accessible to general HTML markup

Reply:
@jasoncypret I fixed it. Can you check it out again?